src/app/add-item/add-item.component.ts

```diff
@@ -58,11 +72,22 @@ export class AddItemComponent implements OnInit {
     }
     const newItem: AddActionItem = {
       title: title.value,
-      projectName: projectName.value.name,
+      projectName: projectName.value,
       type: 'General',
       completed: '0',
       dueDate: dueDate.value
     };
     return newItem;
   }
+  editItem(): void {
+    this.isEditingActionItem = true;
```

Comment:
**CONCERN:** Why is it needed to assign here this value? Shouldn't it be known already?